docs/workflows/common-workflow-steps/appshield-scanner.md

```diff
@@ -0,0 +1,75 @@
+tags: [android, ios, mobile, security, scan, appshield-scanner]
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# Appshield App Scanner for Android / iOS
+
+Appshield Scanner starts its analysis by accepting the application file (AAB/APK for Android, IPA for iOS) and starting dynamic runtime tests after initial file and format validations. 
+It is important to note that while many scanning solutions use ARM/x86_64 based emulators and sandbox environments, Appshield performs all these dynamic tests in **real/physical Android and/or iOS devices.**
+At the end of the dynamic test session, Appshield indicates which security measures/hardenings are present and implemented in the app. If some testcases were failed due to unforeseen errors or cannot be tested various bypass mechanisms 
+implemented by the app itself, Appshield then starts a static, AI-powered analysis for mentioned testcases to gather additional findings and then reaches a final verdict.
+
+**Security measures/hardenings Appshield Scanner checks for in an application includes (but not limited to)**:
+
+* **Root/Jailbreak Detection**: Detects whether the device has been rooted or jailbroken, which increases the risk of unauthorized system access and tampering.
+* **Frida / Anti-Hooking Detection:** Identifies runtime instrumentation frameworks (like Frida) or hooks that intercept and modify function calls to prevent dynamic analysis and tampering.
+* **Anti Code Injection Detection:** Detects attempts to inject or load unsigned/native code or libraries into the app’s process to prevent unauthorized code execution.
+* **Anti Debugging Detection:** Detects debuggers or debugger-like behaviors (ptrace, debug flags, wait loops) to prevent runtime inspection and breakpoints.
+* **Screenshot / Screen Recording Detection:** Detects and blocks or flags attempts to capture the app’s UI via screenshots or screen recording to protect sensitive UI content.
+* **Screen Mirroring Detection:** Detects when the app’s display is being mirrored to another device (e.g., via AirPlay or casting) to prevent remote observation of sensitive data.
+* **Keylogger Detection:** Detects behaviors or hooks that capture keystrokes or input events to prevent credential or data exfiltration.
+* **Tapjacking Detection:** Detects overlay or UI manipulation attacks that trick users into tapping hidden controls or approving actions they didn’t intend.
+
+### Prerequisites
+
+Before running the **Appshield Scanner** step, you must complete certain prerequisites, as detailed in the table below:
+
+| Prerequisite Workflow Step                                                                         | Description                                                                                                                                                              |
```

Comment:
Could you please add same Prerequisites step for iOS side. 
Like this doc: https://docs.appcircle.io/workflows/common-workflow-steps/data-theorem-mobile-secure